lobster/tools/codebeamer/codebeamer.py

```diff
@@ -522,17 +534,24 @@ def main():
 
     elif cb_config.get("import_query"):
         try:
-            query_id = int(cb_config["import_query"])
-        except ValueError:
-            sys.exit("lobster-codebeamer: Query ID must be an integer.")
-        if query_id < 1:
-            sys.exit("lobster-codebeamer: Query ID must be a positive integer.")
+            if isinstance(cb_config["import_query"], int):
+                cb_config["import_query"] = int(cb_config["import_query"])
+                if cb_config["import_query"] < 0:
+                    ap.error("import-query must be a positive integer")
+            elif isinstance(cb_config["import_query"], str):
+                if (cb_config["import_query"].startswith("-") and
+                    cb_config["import_query"][1:].isdigit()):
+                    ap.error("import-query must be a positive integer")
+                elif cb_config["import_query"].startswith("-"):
+                    ap.error("import-query must be a valid cbQL query")
```

Comment:
```suggestion
            if ((isinstance(cb_config["import_query"], int) and int(cb_config["import_query"])<0)
                 or (isinstance(cb_config["import_query"], str) and cb_config["import_query"].startswith("-"))):
                 ap.error("import-query must be a valid codebeamer report id or a valid cbQL query ")
     
```

Reply:
This code won't work as it is
What if the user provides input like below
`import_query: "271679129"`

This input is then considered as a "query string"

Reply:
okay got it.